apps/src/feedback.js

```diff
@@ -919,24 +916,24 @@ FeedbackUtils.prototype.getGeneratedCodeDescription = function (codeInfoMsgParam
  *        to display instead of the usual show code description
  */
 FeedbackUtils.prototype.showGeneratedCode = function(Dialog, appStrings) {
-  var codeDiv = this.getGeneratedCodeElement_({
+  var codeDiv = document.createElement('div');
+
+  var generatedCodeComponent = this.getGeneratedCodeComponent_({
     generatedCodeDescription: appStrings && appStrings.generatedCodeDescription
   });
 
-  var buttons = document.createElement('div');
-  ReactDOM.render(React.createElement(DialogButtons, {
-    ok: true
-  }), buttons);
-  codeDiv.appendChild(buttons);
+  ReactDOM.render(<ShowCode
+    generatedCodeComponent={generatedCodeComponent}
```

Comment:
Actually, this component is extremely simple. How would we feel about inlining the whole thing?

``` javascript
ReactDOM.render(<div>
  {generatedCodeComponent}
  <DialogButtons ok={true} />
</div>, codeDiv);
```